_data/bib_stats_msc.yml

```diff
@@ -3,7 +3,7 @@
 # Area/Primary/Secondary
 
 - code: "00"
-  name: General
+  name: General and overarching topics; collections
```

Comment:
Are these changes related to the change in convbib.g?

Reply:
No - as 70f4a8861cf7bc4293d1f8647f2319a2b8cc1b15 commit says, that should be done earlier with fd77e0b9379604e14d3fd9e837df81bc4bd7b693, but that PR was already merged.